opal/mca/event/libevent2021/libevent2021_component.c

```diff
@@ -120,12 +120,14 @@ const opal_event_component_t mca_event_libevent2021_component = {
     }
 };
 
+#define BUFF_SIZE 1024
```

Comment:
IIRC, the number of eventops is something like 5, and each string is on the order of 4-5 chars. So even 256 should be more than enough.